examples/classic-slides/index.html

```diff
@@ -246,22 +250,32 @@ <h1>Motion effects 101</h1>
         </div>
     </div>
 
-    <div id="addons" class="step slide title" data-rel-x="-600" data-rel-y="2200" data-rotate="120">
+    <div id="zoom" class="step" data-rel-reset data-rel-x="-0.25w" data-rel-y="0.5h" data-scale="0.5">
+        <div class="notes">
+            <p>This step zoom in to left bottom of previous slide to see to small text.</p>
+            <p>It's a empty and transparent.</p>
+            <p><code>data-rel-reset</code> is used to prevent this step from inheriting the relative positioning from previous slide.</p>
+        </div>
+    </div>
+
+    <div id="addons" class="step slide title" data-rel-to="motions" data-rel-inherit="motions">
```

Comment:
Would rel-to and rel-inherit ever be different?

Should we say that inherit implies also rel-to? I think we're missing something in the design here still, but I keep forgetting details we discussed months ago. Sorry.

I'm also thinking you introduced `data-rel-position="relative"` mode. Would it make sense if the above line would instead just be:

    <div id="addons" class="step slide title" data-rel-to="motions" data-rel-position="relative">

...and do exactly the same thing?

(Of course, data-rel-position could also be omitted, since it is itself inherited.)

I think my point here is that you are introducing a new attribute to keep existing syntax backward compatible, but `data-rel-position="relative"` is already such new syntax where you can define behavior without worrying about backward compatibility.